source/user-manual/api/configuration.rst

```diff
@@ -253,6 +255,8 @@ cache
 Security configuration options
 ------------------------------
 
+This section contains all the options available for the ``/var/ossec/api/security/security.yaml`` configuration file.
```

Comment:
The correct path is: {WAZUH_PATH}/api/configuration/security/security.yaml

Reply:
Done